doc/source/examples/index.rst

```diff
@@ -6,28 +6,28 @@ Examples
 Here are examples using Mechanical with the ``ansys-mechanical-core`` library through
 an embedded instance or remote session.
 
-.. grid:: 2
+.. grid:: 1 2 2 2
 
    .. grid-item-card:: Embedding Examples
-      :link: https://embedding.examples.mechanical.docs.pyansys.com/index.html
+      :padding: 2 2 2 2
+      :link: https://embedding.examples.mechanical.docs.pyansys.com/examples/index.html
       :text-align: center
-      :margin: 2 2 0 0
 
       Uses PyMechanical to embed an instance of Mechanical directly within Python as a Python object.
 
    .. grid-item-card:: Remote Session Examples
-      :link: https://examples.mechanical.docs.pyansys.com/index.html
+      :padding: 2 2 2 2
+      :link: https://examples.mechanical.docs.pyansys.com/examples/index.html
       :text-align: center
```

Comment:
Recommend removing center alignment based on @Revathyvenugopal162's feedback and documentation team's feedback